build.hya.sh

```diff
@@ -0,0 +1,28 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+export JAVA_HOME=/usr/lib/jvm/java-8-openjdk-amd64
+echo "set jdk8 finished"
+mvn=/home/hya/software/ideaIU-2019.2.3/idea-IU-192.6817.14/plugins/maven/lib/maven3/bin/mvn
```

Comment:
This is only for hya, since my default JAVA_HOME is java 11 and I use maven provided by idea